Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -630,32 +619,17 @@ - (NSUInteger)maxBatchSize
     return 100;
 }
 
-- (NSURL *)userIDURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.userId");
-}
-
-- (NSURL *)queueURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.queue.plist");
-}
-
-- (NSURL *)traitsURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.traits.plist");
-}
-
 - (NSString *)getUserId
 {
-    return [[NSUserDefaults standardUserDefaults] valueForKey:SEGUserIdKey] ?: [[NSString alloc] initWithContentsOfURL:self.userIDURL encoding:NSUTF8StringEncoding error:NULL];
+    return [[NSUserDefaults standardUserDefaults] valueForKey:SEGUserIdKey] ?: [self.analytics.storage stringForKey:kSEGUserIdFilename];
 }
 
 - (void)persistQueue
 {
 #if TARGET_OS_TV
-    [[NSUserDefaults standardUserDefaults] setValue:[self.queue copy] forKey:SEGQueueKey];
+    [self.analytics.storage setArray:[self.queue copy] forKey:SEGQueueKey];
```

Comment:
I think these macros should be handled in the SEGFileStorage class - I thought that was one the reasons of the SEGFileStorage abstraction. 

It could probably be just called SEGStorage now that I think about it since it'd deal with NSUserDefaults and files.